Source/WebKit/Platform/cocoa/ExtensionCapability.h

```diff
@@ -31,21 +31,34 @@
 #include <wtf/RetainPtr.h>
 
 OBJC_CLASS BEProcessCapability;
+OBJC_CLASS _SECapability;
 
 namespace WebKit {
 
+using PlatformCapability = std::variant<RetainPtr<BEProcessCapability>, RetainPtr<_SECapability>>;
+
 class ExtensionCapability {
 public:
     virtual ~ExtensionCapability() = default;
     virtual String environmentIdentifier() const = 0;
-    BEProcessCapability *platformCapability() const { return m_platformCapability.get(); }
+    PlatformCapability platformCapability() const { return m_platformCapability; }
```

Comment:
Should return a `const PlatformCapability&`

Reply:
Ah, good catch, will fix!